README.md

```diff
@@ -113,6 +113,13 @@ $ cd module-name (e.g: pulsar-client)
 $ mvn test -Dtest=unit-test-name (e.g: ConsumerBuilderImplTest)
 ```
 
+Run Selected Test packages:
+
+```bash
+$ cd module-name (e.g: pulsar-broker)
+$ mvn test -pl module-name -Dinclude=org/apache/pulsar/**/*.java
```

Comment:
@aahmed-se - In the docs, since there is a cd into 'module-name' do we still need the -pl option?
If we are using the -pl option,  using the -am ([--also-make](https://books.sonatype.com/mvnref-book/reference/_using_advanced_reactor_options.html#_specifying_a_subset_of_projects)) would be advisable